python/cugraph/tests/test_connectivity.py

```diff
@@ -142,7 +142,17 @@ def test_weak_cc(graph_file):
 
     # Compare vertices of largest component
     nx_vertices = sorted(lst_nx_components[0])
-    cg_vertices = sorted(lst_cg_components[0])
+    first_vert = nx_vertices[0]
+
+    idx = -1
+    for i in range(len(lst_cg_components)):
+        if first_vert in lst_cg_components[i]:
+            idx = i
+
+    assert idx != -1
```

Comment:
Maybe add some kind of message like this to the assert to help the user understand the failure (and self-document)?
```
assert idx != -1, "Check for Nx vertex in cuGraph results failed"
```

Reply:
fixed